src/components/state-details.js

```diff
@@ -389,7 +384,7 @@ export default function StateDetailsPage ({ location, data }) {
                 />
               </div>
               <div className="col h3">
-                <p className="h2 mt-2 mb-0">
+                <p className="mt-2 mb-0">
```

Comment:
A lot of my changes were removing `h2` and `h3` from paragraphs. We shouldn't individually be setting the size of these as it made nearly impossible to keep them consistent on such a large page. I instead moved this to CSS and only use classes to change font size when a paragraph is supposed to be smaller or larger than normal for this page (`.h2`)